File: 351004/Purenok/Task310/src/main/java/com/example/Task310/bean/Post.java
package com.example.Task310.bean;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "tbl_post", schema = "distcomp")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false, length = 2048)
    private String content;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "story_id", nullable = false)
    private Story story;
}

File: 351002/vorobei/Publisher/DataAccess/Models/Post.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace DataAccess.Models
{
    public class Post : BaseEntity
    {
        [Column("story_id")]
        public int StoryId { get; set; }

        [Column("content")]
        public string Content { get; set; } = string.Empty;
    }
}
